compare/meson.build

```diff
@@ -0,0 +1,19 @@
+compare_src = files(
+    'compare.c',
+    'image.c',
+)
+
+libass_compare = executable(
+    'libass_compare',
```

Comment:
`libass_test` is only namespaced because meson reserves the `test` name; does the same apply here?